tests/outputs.test.ts

```diff
@@ -0,0 +1,34 @@
+import * as Nexus from 'nexus'
+import { generateSchemaAndTypesWithoutThrowing } from './__utils'
+
+it('only publishes output types that do not map to prisma models', async () => {
+  const datamodel = `
+  model User {
+    id  Int @id
+    name String
+  }
+`
+  const Query = Nexus.objectType({
+    name: 'Query',
+    definition(t: any) {
+      t.crud.user()
+    },
+  })
+
+  const Mutation = Nexus.objectType({
+    name: 'Mutation',
+    definition(t: any) {
+      t.crud.updateManyUser({ filtering: true })
+    },
+  })
+
+  const {
+    schema,
+    typegen,
+    missingTypes,
+  } = await generateSchemaAndTypesWithoutThrowing(datamodel, [Query, Mutation])
+
+  expect(Object.keys(missingTypes)).toContain('User')
```

Comment:
Can we strengthen this to a "same members" check (order does not matter). It should be an error if `BatchPayload` shows up here for example.